index.html

```diff
@@ -44,7 +49,8 @@ <h2 class="connect-link">LET’S CONNECT</h2>
         </section>
         <section class="card">
             <article class="card-section">
-                <img src="./images/tonic-img-1.png" alt="tonic-snapshot" class="tonic-img">
+                <img src="./images/Snapshoot-Portfolio-4.png" alt="tonic-snapshot" class="tonic-img-mobile">
+                <img src="./images/Snapshoot-Portfolio-4.png" alt="tonic-snapshot" class="tonic-img-desktop">
                 <article class="card-section-text">
                     <h2 class="heading-title">
                         Tonic
```

Comment:
- [ ] 🚧 You did a great job with your cards! However, I would suggest you add more spacing between the elements inside:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/213549386-081ea648-863d-4557-b80e-39933321c9ca.png) | ![image](https://user-images.githubusercontent.com/77069011/213549325-5c12832c-e25b-4163-b7df-2deddc0ff107.png) |

- [ ] 🚧 For the mobile version, the images are not showing, and the section is not aligned due to the navigation links being displayed. Kindly look into this and fix it! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/213549710-f6b713a0-b625-4526-bdc5-f03f6b2a1f83.png) | ![image](https://user-images.githubusercontent.com/77069011/213549643-143d6bb3-c455-4595-a6ad-757d4c986e94.png) |